RNTester/e2e/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "setupTestFrameworkScriptFile" : "./test-init.js",
+  "setupFilesAfterEnv" : ["./test-init.js"],
```

Comment:
This is a Jest change that was printing a warning

Reply:
Can you do that in a separate commit?